todo_app/templates/index.html

```diff
@@ -18,13 +18,13 @@ <h2>Items</h2>
               <th>status</th>
               <th>name</th>
           </tr>
-          {%for obj in data%}
+          {%for item in view_model.items%}
```

Comment:
Looks like you're treating all the lists together here (by just using `.items` rather than e.g. `.done_items`, which is fine but worth explicitly noting that it means the unit tests are testing functions that aren't actually used by our app, so they're useful as examples but aren't actually checking any useful business logic!